test/test_markdown.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import warnings
+import json
+import pytest
+from jsonschema import validate
+import commonmark
+
+
+parser = commonmark.Parser()
+
+
+with open('element-schema.json') as fp:
+    SCHEMA = json.load(fp)
+
+
+def test_markdown(markdown_file):
+    if markdown_file.samefile('docs/migration.md'):
+        pytest.skip('migrate.md intentionally contains invalid API Elements')
+
+    with open(markdown_file) as fp:
+        ast = parser.parse(fp.read())
```

Comment:
`markdown_file` is a Path instance, so you can do `markdown_file.read_bytes()` or `markdown_file.read_string()`

Reply:
@honzajavorek In this case yes, but I believe you could invoke the function with pre-defined arguments from CLI using pytest so it may be a string.

Reply:
Ok got to testing this. The following command shows all the setup values of fixtures (markdown_file):

```
$ pipenv run pytest --fixtures-per-test
```

The following can be used to set them:

> [pytest] ini-options in the first pytest.ini|tox.ini|setup.cfg file found:
> - usefixtures (args)       list of default fixtures to be used with this project

Didnn't find any option to provide it as a CLI option, would have been neat for iterating over single with with editor integrations.